### ISSUE TITLE
Import cookie-cutter from NSLS-II and remove unused files

### ISSUE THREAD

**Issue by Sparks29032** [Billingegroup/diffpy-cookiecutter#6]
@8bitsam @Tieqiong Our first course of action should be to copy the NSLS-II and remove all unused files before we modify the existing files to follow our `diffpy.utils` standard. Can someone make a new PR that only copies the NSLS-II cookie-cutter. If you want to make an additional commit after putting in that PR that removes files that are not used in our template, that would be great as well.

Files to remove:

- [ ] setup.cfg
- [ ] versioneer.py

That should be all that is in that PR. Once that is merged, we can update our new PRs onto that initial merge.

Also, make sure your PRs cover one or two files at a time. It will be easier to merge and ensure we do not have overlap with who is doing what.

**Comment by 8bitsam** [comment]
I think we have two of these now, PR #7 and #8, need to decide which one to use.

**Comment by Tieqiong** [comment]
lmao we should do this together to avoid extra works

**Comment by Sparks29032** [comment]
I think for the initial merge, we should put in #7 since #8 has additional changes that we should review (such as using `github.repository` to get the repository name information) before we merge.
